docs/serverless/06-01-function-cr.md

```diff
@@ -75,57 +75,62 @@ spec:
+| **status.conditions.type**               | Not applicable | Describes a substage of the Function CR processing. There are three condition types that a Function has to meet to be ready: `ConfigurationReady`, `BuildReady`, and `Running`. When displaying the Function status in the terminal, these types are shown under `CONFIGURED`, `BUILT`, and `RUNNING` columns respectively. All condition types can change asynchronously depending on the type of Function modification, but all three need to be in the `True` status for the Function to be considered successfully processed. |
+
+| Reason                           | Type                 | Description                                                                                                               |
+| -------------------------------- | -------------------- | ------------------------------------------------------------------------------------------------------------------------- |
+| `ConfigMapCreated`               | `ConfigurationReady` | A new ConfigMap was created based on the Function CR definition.                                                          |
+| `ConfigMapUpdated`               | `ConfigurationReady` | The existing ConfigMap was updated after changes in the Function CR name, its source code or dependencies.                |
+| `ConfigMapError`                 | `ConfigurationReady` | The ConfigMap could not be created or updated due to an error.                                                            |
+| `JobFailed`                      | `BuildReady`         | The image with the Function's configuration could not be created due to an error.                                         |
+| `JobCreated`                     | `BuildReady`         | The Kubernetes Job resource that builds the Function image was created.                                                   |
+| `JobRunning`                     | `BuildReady`         | The Job is in progress.                                                                                                   |
+| `JobsDeleted`                    | `BuildReady`         | Previous Jobs responsible for building Function images were deleted.                                                      |
+| `JobFinished`                    | `BuildReady`         | The Job was finished and the Function's image was uploaded to the Docker Registry.                                        |
+| `DeploymentCreated`              | `Running`            | A new Deployment referencing the Function's image was created.                                                            |
+| `DeploymentUpdated`              | `Running`            | The existing Deployment was updated after such changes as the Function's image, scaling parameters, variables, or labels. |
```

Comment:
```suggestion
| `DeploymentUpdated`              | `Running`            | The existing Deployment was updated after changing the Function's image, scaling parameters, variables, or labels. |
```

Reply:
(I know it was already there, but looking at it now, I would put it this way ;) )